wikipedia-sentences/README.md

```diff
@@ -154,13 +189,13 @@ Here is a small overview if you're interested in understanding what each file in
 |File   | Explanation  |
 |--|--|
 |📂 `flows/`  | Folder to store Flow configuration    |
-|--- 📃 `flows/index.yml`  | Contains the details of which executors should be used for indexing your data. |
-|--- 📃 `flows/query.yml`  | Contains the details of which executors should be used for querying your data. |
+|--- 📃 `flows/index.yml`  | Contains the details of which Executors should be used for indexing your data. |
```

Comment:
can you add the incremental index flow as a row here please?

Reply:
Added in https://github.com/jina-ai/examples/pull/577/commits/ea5f5db0d2e324bcc751aa8cb62fd4b44e9bad62